forgerock-auth/src/main/java/org/forgerock/android/auth/devicebind/BiometricUtil.kt

```diff
@@ -0,0 +1,53 @@
+package org.forgerock.android.auth.devicebind
+
+import androidx.fragment.app.FragmentActivity
+import org.forgerock.android.auth.InitProvider
+import org.forgerock.android.auth.biometric.AuthenticatorType
+import org.forgerock.android.auth.biometric.BiometricAuth
+import org.forgerock.android.auth.biometric.BiometricAuthCompletionHandler
+import org.forgerock.android.auth.callback.DeviceBindingAuthenticationType
+
+open class BiometricUtil {
+
+    private lateinit var biometricAuth: BiometricAuth
+    private var authenticationtype: DeviceBindingAuthenticationType? = null
+
+    @JvmOverloads
+     fun getBiometricAuth(biometricListener: BiometricAuthCompletionHandler,
+                                   fragmentActivity: FragmentActivity = InitProvider.getCurrentActivityAsFragmentActivity(),
+                                   titleValue: String,
+                                   subtitleValue: String,
+                                   authentication: DeviceBindingAuthenticationType,
+                                   descriptionValue: String): BiometricAuth {
+
+
+        val authenticationType = if(authentication == DeviceBindingAuthenticationType.BIOMETRIC_ALLOW_FALLBACK)
+            AuthenticatorType.STRONG_OR_DEVICE_PIN else AuthenticatorType.STRONG
+
+        val deviceCredential = authentication == DeviceBindingAuthenticationType.BIOMETRIC_ALLOW_FALLBACK
+        biometricAuth = BiometricAuth(titleValue,
+            subtitleValue,
+            deviceCredential,
+            fragmentActivity,
+            biometricListener,
+            descriptionValue,
+            authenticationType)
+        authenticationtype = authentication
+        return biometricAuth
+    }
+
+    fun isSupported(): Boolean {
```

Comment:
Can u provide the logic to determine the device support?

Reply:
Done